### PR TITLE
Show added books on /mybooks page

### PR DESCRIPTION
**Pull request by sacidcelik**
Thanks for reviewing :)

This feature branch adds functionality to display added books (adding via search on /home) on the /mybooks page. Removing books on search, also removes books from the /mybooks page.

If you want to try this locally, starting the server via `npm run dev` from root should work if you remove the api key from the fetch function request in the backend (server/controller/search.controller.js):

`const searchAddress = 'https://www.googleapis.com/books/v1/volumes?key=${apiKey}&maxResults=25&q=';` -> `const searchAddress = 'https://www.googleapis.com/books/v1/volumes?maxResults=25&q=';`

### CODE REVIEW

**Comment by felixcanditt** [comment]
understandable guideline for what the reviewing person should check and set up :) checking now

**Comment by felixcanditt** [comment]
feature works in my browser as described

File: client/src/components/Library.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      {library.map((book) => {
+        return (
+          <BookCard key={book.id} data-testid="library-book">
+            <img
+              src={book.volumeInfo.imageLinks.thumbnail}
+              alt="Book Cover"
+              width="102.4"
+              height="159.2"
+            />
+            <div>
```
Comment (felixcanditt) [inline client/src/components/Library.js:16]: do you need div and p here? maybe one is enough?

**Comment by felixcanditt** [comment]
looks good to me, ready to merge :)

File: client/src/components/Library.js (hunk 1 above)
Comment (sacidcelik) [reply client/src/components/Library.js:16]: changed it to <p> only without the additional <div>

**Pull request merged**